.github/workflows/no-frontend-release.yml

```diff
@@ -12,22 +12,6 @@ jobs:
         with:
           ref: master
 
-      - name: Merge master to no-frontend
-        uses: devmasx/merge-branch@v1.3.1
-        with:
-          type: now
-          target_branch: 'no-frontend'
-          github_token: ${{ github.token }}
-
-      - uses: actions/checkout@v2
-        with:
-          ref: no-frontend
-
-      - uses: actions/setup-java@v2
-        with:
-          distribution: 'adopt'
-          java-version: 8
-
```

Comment:
We can use the master branch directly to build the no-frontend image. 
The command `mvn -ntp package -P production ...` takes in a profile arg that determines whether or not frontend is built.

Reply:
Nice! That'll simplify everything.